alert-policies/apache-flink/JobManagerHeapMemoryUsage.yml

```diff
@@ -0,0 +1,26 @@
+name: Job Manager Heap Memory Usage
+
+description: |+
+  This alert is triggered when the job manager - heap memory usage is higher than 10mb during 5 minutes.
```

Comment:
I feel there is no need of using '-' in the text. Request you to kindly remove it.

Reply:
Required changes are done.